src/test/java/org/assertj/core/api/bytearray/ByteArrayAssert_containsExactlyInAnyOrder_with_Byte_array_Test.java

```diff
@@ -0,0 +1,43 @@
+package org.assertj.core.api.bytearray;
+
+import org.assertj.core.api.ByteArrayAssertBaseTest;
+import org.assertj.core.api.ByteArrayAssert;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.catchThrowable;
+import static org.assertj.core.api.BDDAssertions.then;
+import static org.assertj.core.error.ShouldNotBeNull.shouldNotBeNull;
+import static org.assertj.core.test.ByteArrays.arrayOf;
+import static org.mockito.Mockito.verify;
+
+/**
+ * Tests for <code>{@link ByteArrayAssert#containsExactlyInAnyOrder(Byte[])}</code>.
+ *
+ * @author Lucero Garcia
+ */
+@DisplayName("ByteArrayAssert containsExactlyInAnyOrder(Byte[])")
+class ByteArrayAssert_containsExactlyInAnyOrder_with_Byte_array_Test extends ByteArrayAssertBaseTest {
+
+  @Test
+  void should_fail_if_values_is_null() {
+    // GIVEN
+    Byte[] values = null;
+    // WHEN
+    Throwable thrown = catchThrowable(() -> assertions.containsExactlyInAnyOrder(values));
+    // THEN
+    then(thrown).isInstanceOf(NullPointerException.class)
+      .hasMessage(shouldNotBeNull("values").create());
```

Comment:
Please apply AssertJ formatting rules:
```suggestion
                .hasMessage(shouldNotBeNull("values").create());
```